docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -402,18 +402,116 @@ SELECT
 └───────────────────────────┴──────────────────────────────┘
 ```
 
-## parseDateTimeBestEffort {#type_conversion_functions-parsedatetimebesteffort}
+## parseDateTimeBestEffort {#parsedatetimebesteffort}
 
-Parse a number type argument to a Date or DateTime type.
-different from toDate and toDateTime, parseDateTimeBestEffort can progress more complex date format.
-For more information, see the link: [Complex Date Format](https://xkcd.com/1179/)
+Converts a date and time in the [String](../../data_types/string.md) representation to [DateTime](../../data_types/datetime.md#data_type-datetime) data type.
+
+The function parses [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), [RFC 1123 - 5.2.14  RFC-822 Date and Time Specification](https://tools.ietf.org/html/rfc1123#page-55), ClickHouse's and some other date and time formats.
+
+
+**Syntax**
+
+```sql
+parseDateTimeBestEffort(s[, time_zone]);
+```
+
+**Parameters**
+
+- `s` — String containing a date and time to convert. [String](../../data_types/string.md).  
+- `time_zone` — Time zone. The function parses `s` according to the time zone. [String](../../data_types/string.md).
+
+
+**Supported non-standard formats**
+
+- A string containing 9..10 digit unixtimestamp.
+- A string with a date and a time component: `YYYYMMDDhhmmss`, `DD/MM/YYYY hh:mm:ss`, `DD-MM-YY hh:mm`, `YYYY-MM-DD hh:mm:ss`, etc.
+- A string with a date but no time component: `YYYY`, `YYYYMM`, `YYYY*MM`, `DD/MM/YYYY`, `DD-MM-YY`, `DD`, etc.
+- A string with a time but no date component:  `hh:mm`, `hhmm`, `hh` - if already have day of month.
+- A string that includes the date and time along with time zone offset information: `2020-12-12 17:36:00 -5:00`, etc.
```

Comment:
```suggestion
- A string that includes date and time along with time zone offset information: `2020-12-12 17:36:00 -5:00`, etc.
```
This one is inconsistent with previous items by showing specific numbers instead of YMDhms combination